media-sound/qmmp/qmmp-2.2.3.ebuild

```diff
@@ -0,0 +1,174 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg
+
+DESCRIPTION="Qt-based audio player with winamp/xmms skins support"
+HOMEPAGE="https://qmmp.ylsoftware.com"
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="
+		https://qmmp.ylsoftware.com/files/qmmp/$(ver_cut 1-2)/${P}.tar.bz2
+		https://downloads.sourceforge.net/project/qmmp-dev/qmmp/$(ver_cut 1-2)/${P}.tar.bz2
+
+	"
+	KEYWORDS="~amd64 ~x86"
+else
+	inherit subversion
+	local BRANCH_VER="2.2" # NOTE: bump this to $(ver_cut 1-2) for live ebuild
+	ESVN_REPO_URI="svn://svn.code.sf.net/p/qmmp-dev/code/branches/qmmp-${BRANCH_VER}"
+fi
+
+LICENSE="CC-BY-SA-4.0 GPL-2+" # default skin & source code
```

Comment:
GPL-2/GPL-2+ is quite obscure to me, but I don’t see any recent commit for the license ?